README.md

```diff
@@ -1,39 +1,58 @@
 # Ecosystem Infra Rotation
 
-This is a tool for the [ecosystem infra](https://bit.ly/ecosystem-infra) rotation.
+This is a tool for the [Ecosystem Infra](https://bit.ly/ecosystem-infra) rotation.
 
 ## Usage
 
 Visit https://ecosystem-infra-rotation.appspot.com/ and follow the instructions.
 
 If you reload often you might hit the GitHub API limit for unauthenticated requests.
-If this happens, [generate a new access token](https://github.com/settings/tokens/new) and pass it in the URL: https://ecosystem-infra-rotation.appspot.com/#GH_TOKEN=abcdef
+If this happens, [generate a new access token](https://github.com/settings/tokens/new)
+and pass it in the URL: https://ecosystem-infra-rotation.appspot.com/#GH_TOKEN=abcdef
+
+This is an AppEngine project, so we assume you already have Google Cloud SDK (including
+Python plugins) set up locally.
 
 ## Running locally
 
-The build script needs Bower, Python 3 and Virtualenv:
+You will need `npm` as well as `python3` and `virtualenv`:
 ```bash
-sudo npm install -g bower
-sudo apt install python3 virtualenv
+sudo apt install python3 virtualenv python3-venv
```

Comment:
This doesn't say how to install `npm`, and are *both* `virtualenv` and `python3-venv` really needed?

Reply:
I saw this too now:
```
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt-get install python3-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.
```

Still worked fine, but it's probably best to install it.

I'm fine with not providing instructions for how to install `npm`, but then the preceding line should be tweaked to not suggest that this is how to install it.